client/my-sites/site-settings/settings-newsletter/main.tsx

```diff
@@ -104,36 +120,99 @@ const NewsletterSettingsForm = wrapSettingsForm( getFormSettings )( ( {
 			{ siteId && <QueryJetpackModules siteId={ siteId } /> }
+			{ /* @ts-expect-error SettingsSectionHeader is not typed and is causing errors */ }
+			<SettingsSectionHeader
+				disabled={ disabled }
+				id="subscriptions"
+				isSaving={ isSavingSettings }
+				onButtonClick={ handleSubmitForm }
+				showButton
+				title={ translate( 'Subscriptions' ) }
+			/>
+			<Card className="site-settings__card">
+				<SubscribeModalSetting
+					disabled={ disabled }
+					handleToggle={ handleToggle }
+					value={ sm_enabled }
+				/>
+			</Card>
+
+			{ /* @ts-expect-error SettingsSectionHeader is not typed and is causing errors */ }
+			<SettingsSectionHeader
+				disabled={ disabled }
+				id="email-settings"
+				isSaving={ isSavingSettings }
+				onButtonClick={ handleSubmitForm }
+				showButton
+				title={ translate( 'Email' ) }
+			/>
+			<Card className="site-settings__card">
+				<FeaturedImageEmailSetting
+					disabled={ disabled }
+					handleToggle={ handleToggle }
+					value={ wpcom_featured_image_in_email }
+				/>
+			</Card>
+			<Card className="site-settings__card">
+				<ExcerptSetting
+					disabled={ disabled }
+					updateFields={ updateFields }
+					value={ wpcom_subscription_emails_use_excerpt }
+				/>
+			</Card>
+
 			{ ( isSimpleSite || isAtomicSite || hasNewsletterCategoriesBlogSticker ) && (
-				<form onSubmit={ handleSubmitForm }>
+				<>
+					{ /* @ts-expect-error SettingsSectionHeader is not typed and is causing errors */ }
+					<SettingsSectionHeader
+						id="newsletter-categories-settings"
+						title={ translate( 'Newsletter categories' ) }
+						showButton
+						onButtonClick={ handleSubmitForm }
+						disabled={ disabled }
+						isSaving={ isSavingSettings }
+					/>
 					<NewsletterCategoriesSection
 						disabled={ disabled }
 						newsletterCategoryIds={
 							fields.wpcom_newsletter_categories || defaultNewsletterCategoryIds
 						}
 						newsletterCategoriesEnabled={ fields.wpcom_newsletter_categories_enabled }
 						handleToggle={ handleToggle }
-						handleSubmitForm={ handleSubmitForm }
 						updateFields={ updateFields }
-						isSavingSettings={ isSavingSettings }
 					/>
-				</form>
+				</>
 			) }
-			<form onSubmit={ handleSubmitForm }>
-				<NewsletterSettingsSection
-					fields={ fields }
-					handleToggle={ handleToggle }
-					handleSubmitForm={ handleSubmitForm }
+
+			{ /* @ts-expect-error SettingsSectionHeader is not typed and is causing errors */ }
+			<SettingsSectionHeader
+				disabled={ disabled }
+				id="messages"
+				isSaving={ isSavingSettings }
+				onButtonClick={ handleSubmitForm }
+				showButton
+				title={ translate( 'Messages' ) }
+			/>
+			<Card className="site-settings__card">
+				<EmailsTextSetting
 					disabled={ disabled }
-					isSavingSettings={ isSavingSettings }
-					savedSubscriptionOptions={ savedSubscriptionOptions }
 					updateFields={ updateFields }
+					value={ subscription_options }
 				/>
-			</form>
```

Comment:
I see we're moving from two form elements in the old code to one in the new code. I assume that's fine (and it tests fine), but I was wondering why those were originally separated?

Reply:
I think it was just the side-effect how development — The Categories bit was quite well isolated from the rest of the settings on this page, which is fair.

I don't think each section needs its own form, and we can simplify by submitting all changes on the page from all "Save" buttons instead.

Similarly I moved the header component out of categories to the main component — undoing a bit of the clear boundary/isolation but I think it makes files overall easier to parse and more consistent.